causal_testing/testing/causal_test_result.py

```diff
@@ -27,7 +27,7 @@ def __init__(
         self,
         estimator: Estimator,
         test_value: TestValue,
-        confidence_intervals: [float, float] = None,
+        confidence_intervals: [pd.Series, pd.Series] = None,
```

Comment:
Would it make sense to switch to two instance variables, `ci_low` and `ci_high`?

Reply:
I think this will make accessing the confidence intervals clearer. I've made it an issue to hopefully follow up on in another PR:
https://github.com/CITCOM-project/CausalTestingFramework/issues/266